lm_eval/tasks/tmmluplus/READMD.md

```diff
@@ -0,0 +1,664 @@
+---
```

Comment:
Could you make this readme align with the template README here: https://github.com/EleutherAI/lm-evaluation-harness/blob/main/templates/new_yaml_task/README.md

The main things we want are:
- The paper's title, abstract, and BibTeX citation
- Bibtex citations for any datasets used as components by the authors for this aggregated benchmark
- a list of the task and group names defined for this task, e.g. group: `tmmluplus`, `tmmluplus_stem`, ..., tasks: `tmmluplus_administrative_law`, ....
- A note about whether this matches the results from https://github.com/iKala/ievals on a tested model would be also very helpful!